helpdesk-bot/docker/k8s_deployment.yaml.template

```diff
@@ -0,0 +1,80 @@
+kind: Deployment
+apiVersion: extensions/v1beta1
+metadata:
+  name: helpdesk-bot
+  namespace: default
+  labels:
+    run: helpdesk-bot
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        run: helpdesk-bot
+    spec:
+      containers:
+        - name: helpdesk-bot
+          image: gcr.io/sym-dev-plat/helpdesk-bot:<VERSION>
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 8080
+              protocol: TCP
+          env:
+            - name: SERVER_PORT
+              value: '8080'
+            - name: HELPDESK_BOT_HOST
+              value: helpdesk-bot-service
+            - name: HELPDESK_BOT_PORT
+              value: '80'
+            - name: HELPDESK_SERVICE_HOST
+              value: helpdesk-api-service
+            - name: HELPDESK_SERVICE_PORT
+              value: '80'
+            - name: AGENT_HOST
+              value: nexus2-2.symphony.com
+            - name: AGENT_PORT
+              value: '443'
+            - name: SESSION_AUTH_HOST
+              value: sym-nexus2-dev-chat-glb-3-ause1-all.symphony.com
```

Comment:
All the service endpoints, username, keystores, passwords, and streamId are hard-coded. Shouldn't it be configured externally via environment variables or something like that?

Reply:
This file is not part of the Dcoker image. It is used as a deployment descriptor on kubernetes. There must be one of this file for each environment. By the way, these pair of name/value become environment variables when the container is running.